From e0e43e93777701f5f2e6f7505b8e9b9e9681aee1 Mon Sep 17 00:00:00 2001
From: Panu Matilainen <pmatilai@redhat.com>
Date: Wed, 29 May 2024 12:57:34 +0300
Subject: [PATCH] Fix regression on subpackage debuginfo RPMTAG_SOURCERPM
 missing

Didn't bisect but this is probably due to mucking around with the
src.rpm name generation in dc47a50c6345a25b861305d8aa8ae464098834ff,
but it's also much saner this way: the info should be copied from the
main debuginfo package along with all the other similar stuff.
---
 build/files.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/build/files.c b/build/files.c
index f57d183d5..f85ad7ed0 100644
--- a/build/files.c
+++ b/build/files.c
@@ -2805,6 +2805,7 @@ static rpmTagVal copyTagsFromMainDebug[] = {
     RPMTAG_OS,
     RPMTAG_PLATFORM,
     RPMTAG_OPTFLAGS,
+    RPMTAG_SOURCERPM,
     0
 };
 
-- 
2.45.1

